### DISCUSSION TITLE
Live telemetry and race control windows not able to align with hotkeys

### DISCUSSION THREAD

**Discussion by IvoElferink** [f1multiviewer/issue-tracker#255] [Feature requests]
I'm using [magnet](https://magnet.crowdcafe.com/) to align my macOS windows, but unfortunately the windows of the live telemetry and race control are not responding to the hotkeys. It is possible to align the windows with the mouse, but that is not quick and easy.

Would it be possible to make the windows respond the the hotkeys of for example magnet?

**Comment by f1multiviewer** [comment]
Hey, this is a known issue. I'm not quite sure why this happens. Do other windows (like the main window) work? I'm not quite sure what APIs magnet is using, so it's hard for me to say exactly what is the cause of this.

**Reply by IvoElferink** [reply]
All other windows, main window and streams are working with the hotkeys, no issues there. It's only the three live timing/map/race control windows that are not responding to the hotkeys.

**Reply by f1multiviewer** [reply]
It's possible that this is due to those windows not being allowed to minimize, maybe that disables certain APIs used by magnet. I can't really allow that yet, since it'll break the app (background throttling is a bitch) but I'll see what I can do.

**Comment by jeppeolesen** [comment]
This seems to be specific for Magnet - I'm unable to replicate using either Rectangle App or Raycast. Closing this.

If it's still an issue, please file it as a bug over here: https://github.com/f1multiviewer/issue-tracker/issues
